validator_client/src/http_api/ui.rs

```diff
@@ -0,0 +1,127 @@
+use parking_lot::RwLock;
+use serde::{Deserialize, Serialize};
+use std::path::Path;
+use std::sync::Arc;
+use sysinfo::{CpuExt, DiskExt, System, SystemExt};
+
+/// System related health, specific to the UI.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct SystemHealth {
+    /// Total memory of the system.
+    pub total_memory: u64,
+    /// Total free memory available to the system.
+    pub free_memory: u64,
+    /// Total used memory.
+    pub used_memory: u64,
+
+    /// System load average over 1 minute.
+    pub sys_loadavg_1: f64,
+    /// System load average over 5 minutes.
+    pub sys_loadavg_5: f64,
+    /// System load average over 15 minutes.
+    pub sys_loadavg_15: f64,
+
+    /// Total cpu cores.
+    pub cpu_cores: usize,
+    /// Total cpu threads.
+    pub cpu_threads: usize,
+    /// The global cpu frequency.
+    pub global_cpu_frequency: f32,
+
+    /// Total capacity of disk.
+    pub disk_bytes_total: u64,
+    /// Free space in disk.
+    pub disk_bytes_free: u64,
+
+    /// System uptime.
+    pub system_uptime: u64,
+    /// Application uptime.
+    pub app_uptime: u64,
+    /// The System name
+    pub system_name: String,
+    /// Kernel version
+    pub kernel_version: String,
+    /// OS version
+    pub os_version: String,
+    /// Hostname
+    pub host_name: String,
+}
+
+impl SystemHealth {
```

Comment:
Rather than duplicating this, can we move `SystemHealth` to its own crate under `common`?

Reply:
ahaha yeah, i was being lazy with both of the points you've raised. 

Can't sneak anything through :p

Reply:
sorry not sorry :P